cmd/erasure-object.go

```diff
@@ -608,19 +609,30 @@ func (er erasureObjects) putObject(ctx context.Context, bucket string, object st
 
 		// If we have offline disks upgrade the number of erasure codes for this object.
 		parityOrig := parityDrives
+		atomicParityDrives := int64(parityDrives)
```

Comment:
This is AFAICT not enough to have it aligned on all platforms:
```suggestion
		atomicParityDrives := struct{ n int64 }{n: int64(parityDrives)}
```

> The first word in a variable or in an allocated struct, array, or slice can be relied upon to be 64-bit aligned.